tests/lora/conftest.py

```diff
@@ -121,6 +124,14 @@ def sql_lora_files():
     return snapshot_download(repo_id="yard1/llama-2-7b-sql-lora-test")
 
 
+@pytest.fixture(scope="session")
```

Comment:
you don't need this fixture and `TMP_PATH` above. Pytest already has a built in `tmpdir` fixture that you can use https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture